protostar/cli/protostar_arg_type.py

```diff
@@ -33,3 +37,9 @@ def parse_fee_arg_type(arg: str) -> Fee:
     if arg == "auto":
         return arg
     return int(arg)
+
+
+def parse_address_arg_type(arg: str) -> int:
+    if arg.startswith("0x"):
+        return int(arg, 16)
+    return int(arg)
```

Comment:
What is the address convention? Maybe Protostar should tell the user to provide addresses as a hexadecimal number, and other formats won't be supported in the future.

Reply:
Both formats should be accepted, since some tools export the address as an integer

Reply:
Today, I asked the Starknet.py team how should address be represented in the console. They recommended using hex representation. I think we should at least give users an initiative to represent addresses consistently, e.g. by printing a warning. 

https://swmansion.slack.com/archives/C02MBPPR2HY/p1667474916413619

Reply:
We talked about the console output, not the inputs passed to the program

Reply:
I think that's a bit overzealous